### PR TITLE
Integrate LavaMoat allow-scripts protection

### PR DESCRIPTION
**Pull request by weizman**
attempt to add [@lavamoat/allow-scripts](https://github.com/LavaMoat/LavaMoat/tree/main/packages/allow-scripts) protection.

I verified LD works as before by cloning LD on this branch, installing and building all packages successfully

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,5 +36,17 @@
...
+  "dependencies": {
+    "@lavamoat/preinstall-always-fail": "^2.0.0"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "@lavamoat/preinstall-always-fail": false,
+      "@wdio/cli>@wdio/utils>edgedriver": false,
+      "chromedriver": false,
+      "esbuild": false,
+      "geckodriver": false
```
Comment (naugtur) [inline package.json:49]: are you 100% sure the drivers don't need their install scripts? they might need them on a new machine.

**Review by naugtur — APPROVED** [review]
with our current workspace support this is ok.

File: package.json [hunk 2]
```diff
@@ -14,12 +14,13 @@
     "publish": "npm publish --access=public -ws"
```
Comment (naugtur) [inline package.json:14]: I'd suggest adding a setup script that runs install and allow-scripts in a sequence.

Comment (weizman) [reply package.json:14]:
```suggestion
    "publish": "npm publish --access=public -ws",
    "setup": "npm install && npm run allow-scripts"
```

like this?

File: package.json (hunk 1 above)
Comment (weizman) [reply package.json:49]: you are probably right, but if i set these to `true`, what's the benefit I get from allow-scripts eventually?

File: package.json (hunk 2 above)
Comment (naugtur) [reply package.json:14]: Or npm ci depending on your intentions

**Pull request merged**